lib/srfi/%3a176/version-flag.sls

```diff
@@ -0,0 +1,16 @@
+(library (srfi :176 version-flag)
+         (export version-alist)
+         (import (rnrs) (only (mosh config) get-config))
+
+(define (version-alist)
+  `((version ,(get-config "version"))
+      (command "mosh")
+      (scheme.id mosh)
+      (install-dir ,(string-append (get-config "prefix") "/bin"))
```

Comment:
This should probably be just `(install-dir ,(get-config "prefix"))`.

Reply:
Thank you for your review! Done.